### ISSUE TITLE
Bug: External PEM Extensions not loading

### ISSUE THREAD

**Issue by VenaNocta** [RunedUniverse/r4m-maven-extension#9]
PEM Extensions included in other (build-)extensions don't get loaded!

This bug is probably caused by r4m not searching additional extension realms for plexus artifacts

**Comment by VenaNocta** [comment]
Due to the upcoming release I've rechecked this issue and concluded that this will probably never be used in conjunction with build-extensions. Should any other developer require this feature, please open a new issue and link to this one so that we can find a solution.

---

For those wondering why I'm not implementing it right away:

It is pretty much impossible to see build extensions (from r4m's ClassRealm) - the only way to do it would be to build a custom class realm per project when parsing the model where this new realm imports all the 3rd party extensions.

So basically a lot of overhead for sth no one might ever need - well go there when we have to...
